db/seeds.rb

```diff
@@ -27,6 +29,22 @@
 
 puts ">>> SEEDING ENVIRONMENT: #{Rails.env}"
 
+if Region.all.empty?
```

Comment:
I've replaced the "all.empty?" to ".exist?" and -inspired by that change- the ".all.length" to ".count"